docs/conf.py

```diff
@@ -45,7 +46,8 @@
 
 # General information about the project.
 project = u'parso'
-copyright = u'2012 - {today.year}, parso contributors'.format(today=datetime.date.today())
+now = datetime.datetime.utcfromtimestamp(int(os.environ.get('SOURCE_DATE_EPOCH', time.time())))
+copyright = u'2012 - {now.year}, parso contributors'.format(now=now)
```

Comment:
Isn't the until-year in copyright information useless anyway?

Reply:
How do you mean that?